kindelia_core/src/hvm.rs

```diff
@@ -4245,7 +4244,7 @@ pub fn print_io_consts() {
 }
 
 // Serializes, deserializes and evaluates statements
-pub fn test_statements(statements: &Vec<Statement>, debug: bool) {
+pub fn test_statements(network_id: u32, statements: &Vec<Statement>, debug: bool) {
```

Comment:
Maybe here applies the same comment I made on `Node::new`, hmm

Reply:
ok, I will reply to that one.